### DISCUSSION TITLE
How do you create test order with your app in beta?

### DISCUSSION THREAD

**Discussion by shing1001** [amzn/selling-partner-api-models#3352] [General] [answered]
Hello guys, 

I am trying to build an app that would pass the order to another SaaS system. 
My app is in drat status. I have created the OAuth and Redirect etc and also self authorised as a seller. 
But I don't know if I want to create a test order, since the self authorised seller is in prod, how to create a test order that won't actually charge me?
Thanks a lot for anyone's help.

**Comment by littleylv** [comment, marked as answer]
Did you mean [Selling Partner API sandbox](https://developer-docs.amazon.com/sp-api/docs/the-selling-partner-api-sandbox)?

**Reply by ivancortesromero** [reply]
I thought @shing1001 you asked for a real prod-like testing environment as I ask [here](https://github.com/amzn/selling-partner-api-models/discussions/4311). Maybe this other answer helps others.

**Comment by weilinggu** [comment]
Hi Shing1001,

As @ivancortesromero mentioned, please refer to this [discussion](https://github.com/amzn/selling-partner-api-models/discussions/4311)'s answer and follow up there if you have any other questions. Closing this one to avoid duplicated threads going on at the same time.

Thanks!
Weiling
Amazon Selling Partner Services
